pages/in/widget/pay/index.vue

```diff
@@ -679,21 +679,20 @@ export default {
     },
     async submitTransfer() {
       if (this.isChainUpgrading) return;
-      if (this.getAddress && this.getAuthCoreShouldReAuth) {
+      if (this.getAddress && !this.getAuthCoreAccessToken) {
         this.onClickSignInButton();
         return;
       }
       this.isLoading = true;
       this.isSigning = true;
       try {
         const amount = new BigNumber(this.totalAmount);
-        const from = await this.fetchCurrentCosmosWallet();
+        const from = await this.fetchCurrentCosmosWallet() || this.getAddress;
         if (!from) {
           throw new Error('PLEASE_RELOGIN');
         }
         if (!this.isUsingKeplr) {
-          const { cosmosWallet, likeWallet } = this.getUserInfo;
-          if ((cosmosWallet || likeWallet) && from !== cosmosWallet && from !== likeWallet) {
+          if (this.getAddress !== this.getUserInfo.likeWallet) {
```

Comment:
```suggestion
          if (from !== this.getUserInfo.likeWallet) {
```
Would using `from` here work? Since we defined it above anyway